guides/rif-relay/smart-wallets.md

```diff
@@ -16,135 +21,46 @@ There are **two ways** to create a Smart Wallet:
 1. **Regular transaction:** The Requester (or another account on behalf of the Requester) calls the Proxy Factory asking to get a new Smart Wallet. Therefore the Proxy Factory creates a proxy to the SmartWallet code, delegating the ownership to the Requester.
 2. **Sponsored:** It needs to go through the RIF Relay process, which is described in detail below. The requester asks a third party to pay for the Smart Wallet deployment, and the requester pays in tokens for that (or free if it is subsidized by the third-party, a.k.a, Sponsor).
 
+## Send funds
```

Comment:
We should specify that this part may be useful for testing purposes only, if the developers want to use the ERC20 token included in the project

Reply:
updated